docs/source/config.rst

```diff
@@ -541,7 +541,7 @@ Plugins Folder
 TCP Listeners
 -------------
 
-Congfigure the TCP listeners for MQTT, MQTT(SSL) and HTTP Protocols.
+Configure the TCP listeners for MQTT, MQTT(SSL) and HTTP Protocols.
 
 The most important parameter is 'max_clients' - max concurrent clients allowed.
```

Comment:
@emqplus Have you considered using back tics instead of simple quotes to emphasize words more? E.g. like `max_clients`